Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/UserAgendaRepo.kt

```diff
@@ -5,7 +5,7 @@ import android.content.Context.MODE_PRIVATE
 import android.content.SharedPreferences
 import com.mentalmachines.droidcon_boston.utils.SingletonHolder
 
-class UserAgendaRepo private constructor(context: Context) {
+open class UserAgendaRepo private constructor(context: Context) {
```

Comment:
do these have to be opened if you're using Mockito?  I thought it had an override final classes mode using this weird static file hack?

Reply:
I got an error when they weren't open. I'm not sure of an override but I can look into it.